webdriver/tests/bidi/input/set_files/invalid.py

```diff
@@ -0,0 +1,124 @@
+import pytest
+
+from webdriver.bidi.error import NoSuchElementException, UnableToSetFileInputException
+
+pytestmark = pytest.mark.asyncio
+
+
+@pytest.mark.parametrize(
+    "files",
+    [
+        ["path/to/noop.txt"],
+        [],
+    ],
+)
+async def test_set_files_missing_element(
```

Comment:
the invalid tests should follow the `test_params_PARAMNAME_test_name` and `test_params_PARAMNAME_invalid_type` and `test_params_PARAMNAME_invalid_value` for scheme tests

Reply:
Could you elaborate `test_params_PARAMNAME_invalid_type` and `test_params_PARAMNAME_invalid_value`